apps/ledger-live-mobile/src/screens/WalletCentricAsset/index.tsx

```diff
@@ -135,6 +143,49 @@ const AssetScreen = ({ route }: NavigationProps) => {
     }
   }, [currency, navigation]);
 
+  // Dynamic Content Part -------------------
+  const {
+    getAssetCardByIdOrTicker,
+    logDismissCard,
+    logClickCard,
+    logImpressionCard,
+  } = useDynamicContent();
+  const dynamicContentCard = getAssetCardByIdOrTicker(currency);
+
+  const onClickLink = useCallback(
+    (card: ContentCard) => {
```

Comment:
Do we need to pass the content card as a parameter ? We could use `dynamicContentCard` directly instead